app/javascript/controllers/fair_score_landscape_controller.js

```diff
@@ -2,19 +2,25 @@ import { Controller } from "@hotwired/stimulus"
 import { FairScoreChartContainer, FairScoreCriteriaBar } from "../mixins/useFairScore";
 // Connects to data-controller="fair-score-landscape"
 export default class extends Controller {
-  connect() {
+  connect(){
     let fairCriteriaBars = new FairScoreCriteriaBar('ont-fair-scores-criteria-bars-canvas')
     let fairContainer = new FairScoreChartContainer('fair-score-charts-container' , [fairCriteriaBars])
-    let ontologies = jQuery("#ontology_ontologyId");
-
     fairContainer.getFairScoreData("all")
-    ontologies.change( (e) => {
-      if(ontologies.val() !== null){
-        fairContainer.getFairScoreData(ontologies.val().join(','))
-      } else if(ontologies.val() === null){
-        fairContainer.getFairScoreData("all")
-      }
-      e.preventDefault()
-    })
+  }
+  
+  update(event){
```

Comment:
did you remove the equivalent old js code ?

Reply:
Yes, it just require more code to retrieve the values of a tom select in an array